src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/MicroprofileServersAddon.java

```diff
@@ -129,6 +129,12 @@ private String invalidSpecValue(List<String> invalidSpecs) {
     public void adaptMavenModel(Model pomFile, JessieModel model, boolean mainProject) {
 
         String serverName = options.get("server").getSingleValue();
+        // From MP 3.2 on with Helidon, one of the Maven deps changed
+        // So I created a new profile with correct naming.  The solution with the MP version in the profile name is not really useful in this case
+        // since it will be for several MP releases.
+        if ("helidon".equals(serverName) && model.getSpecification().getMicroProfileVersion() == MicroProfileVersion.MP32) {
```

Comment:
@rdebusscher Could we perhaps define a comparator and use "greater or equal"?  I assume they will keep it in 33 too...

Reply:
correct